src/controller.ts

```diff
@@ -12,6 +12,7 @@ import {
 } from "./interlude/interlude";
 import { CheckpointMinigame } from "./minigames/checkpointMinigame";
 import { ScrubMinigame } from "./minigames/scrubMinigame";
+import { ShakingMinigame } from "./minigames/shakingMinigame";
```

Comment:
Do you need to add this to ALL_MINIGAMES?

Reply:
(hint: it's now called MINIGAMES_POOL)